README.md

```diff
@@ -1,17 +1,30 @@
 # Beacons Registration Sprint Boot API
+
 This is an API to allow users to register 406Hz Distress Beacons with the Maritime and Coastguard Agency (MCA).
 
+### Testing
```

Comment:
This can be a sub-heading type, so `##`?

Similarly, you can then change the heading type for the `Running tests` below as well